package.json

```
@@ -84,7 +84,9 @@
     "test:scripts": "cd scripts && NODE_ENV=test mocha --recursive --exit --reporter dot tests",
     "domains:install": "./scripts/local-domains/install",
     "domains:start": "cd ./scripts/local-domains/ && docker-compose up -d",
-    "domains:stop": "cd ./scripts/local-domains/ && docker-compose down"
+    "domains:stop": "cd ./scripts/local-domains/ && docker-compose down",
+    "domains:mac-trust-self-signed-certificate": "cd ./scripts/local-domains/ && docker compose cp caddy:/data/caddy/pki/authorities/local/root.crt /tmp/root.crt && sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain /tmp/root.crt",
+    "domains:linux-trust-self-signed-certificate": "cd ./scripts/local-domains/ && docker compose cp caddy:/data/caddy/pki/authorities/local/root.crt /usr/local/share/ca-certificates/root.crt && sudo update-ca-certificates"
```

Une petite suggestion pour faciliter l'utilisation : 
```suggestion
    "domains:trust-self-signed-certificate:mac": "cd ./scripts/local-domains/ && docker compose cp caddy:/data/caddy/pki/authorities/local/root.crt /tmp/root.crt && sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain /tmp/root.crt",
    "domains:trust-self-signed-certificate:linux": "cd ./scripts/local-domains/ && docker compose cp caddy:/data/caddy/pki/authorities/local/root.crt /usr/local/share/ca-certificates/root.crt && sudo update-ca-certificates"
```

---

J'ai appliqué tes modifs, merci